### PR TITLE
feat(config): add MCOHome MH-S220 FW 3.2

### PR DESCRIPTION
**Pull request by dxfoxd**
New config file for P220 device because parameters have changed from fw version 3.2

PR description here

### CODE REVIEW

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 [hunk 1]
```diff
@@ -0,0 +1,280 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH-S220",
+	"description": "Micro Switch",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255]: Which one is it? S220 or P220? The device IDs look like the S220, the file is called p220 though.
I'm going to assume you meant S220.

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255:1]: the file is missing the `.json` extension.

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+				{
+					"label": "Toggle without specified on/off position",
+					"value": 0
+				},
+				{
+					"label": "Toggle with specified on/off position",
+					"value": 1
+				}
+			]
+		},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255]: Keep these parameters like in the original file.

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+		{
+			"#": "4",
+			"label": "Off-timer",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255]: I don't understand how this label and the options fit together. Maybe there's a better label?

Comment (AlCalzone) [reply packages/config/config/devices/0x015f/mh-p220_3.2-255.255]: Also, params 4 and 5 here look like they are an almost exact match to the old params 5 and 6?

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+					"value": 0
+				},
+				{
+					"label": "Report only ON state of SW1 to AG2",
+					"value": 1
+				},
+				{
+					"label": "Report only OFF state of SW1 to AG2",
+					"value": 2
+				},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255]: Please align the definition of this param with the original one (although that was for switch 2 I think?) and just add the new option 255.

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+				{
+					"label": "Do not send reports to AG2",
+					"value": 255
+				}
+			]
+		},
+		{
+			"#": "9",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255]: This param can be defined very similarly to the previous one. It almost does the same I think, just different options?

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+				{
+					"label": "Do not send reports to AG3",
+					"value": 255
+				}
+			]
+		},
+		{
+			"#": "11",
+			"label": "Send scene ID to AG1 on SINGLE click on S2",
+			"description": "0 = scene activation disabled / 1-255  = send activation CC to association group 1 (AG1) with specified scene ID",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255]:
```suggestion
			"label": "Switch S2: Scene ID (Single Click)",
```

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "12",
+			"label": "Send scene ID to AG1 on DOUBLE click on S2",
+			"description": "0 = scene activation disabled / 1-255  = send activation CC to association group 1 (AG1) with specified scene ID",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255]:
```suggestion
			"label": "Switch S2: Scene ID (Double Click)",
```

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 [hunk 8]
```diff
@@ -0,0 +1,280 @@
...
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "13",
+			"label": "Send scene ID to AG1 on HOLD S2",
+			"description": "0 = scene activation disabled / 1-255  = send activation CC to association group 1 (AG1) with specified scene ID",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255]:
```suggestion
			"label": "Switch S2: Scene ID (Hold)",
```

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 [hunk 9]
```diff
@@ -0,0 +1,280 @@
...
+					"value": 2
+				},
+				{
+					"label": "Disable switch 2 function",
+					"value": 255
+				}
+			]
+		},
+		{
+			"#": "10",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255]: Define this parameter similarly to 8.

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 [hunk 10]
```diff
@@ -0,0 +1,280 @@
...
+		},
+		"2": {
+			"label": "Switch 2",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh-p220_3.2-255.255]: Are you sure these didn't change? The params below seem to indicate those are now
1. Lifeline
2. Switch 1
3. Switch 2

**Review by AlCalzone — CHANGES_REQUESTED** [review]
A couple of remarks.
Also, do you have a manual for the changes?

File: packages/config/config/devices/0x015f/mh-p220_3.2-255.255 (hunk 1 above)
Comment (dxfoxd) [reply packages/config/config/devices/0x015f/mh-p220_3.2-255.255]: My apologies, its the S220.

**Comment by dxfoxd** [comment]
> A couple of remarks. Also, do you have a manual for the changes?

[mh-s220-2021-v3.2.pdf](https://github.com/zwave-js/node-zwave-js/files/11548607/mh-s220-2021-v3.2.pdf)

**Review by AlCalzone — APPROVED** [review]
OK with my changes

**Pull request merged**